install/disconnected_install.adoc

```diff
@@ -195,24 +195,50 @@ endif::[]
 +
 [source, bash]
 ----
+$ docker pull registry.access.redhat.com/openshift3/apb-base:<tag>
+$ docker pull registry.access.redhat.com/openshift3/apb-tools:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-attacher:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-driver-registrar:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-livenessprobe:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/image-inspector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
```

Comment:
These two should be for section "Pull the Red Hat-certified Source-to-Image (S2I) builder images".

Reply:
jenkins-agent-nodejs-8-rhel and jenkins-agent-nodejs-8-rhel?

Reply:
right